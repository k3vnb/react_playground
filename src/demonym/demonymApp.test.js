import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import DemonymApp from './demonymApp'

describe('<DemonymApp /> Component', () => {
    it('renders on load', () => {
        expect(toJson(shallow(<DemonymApp />))).toMatchSnapshot();
    })
})