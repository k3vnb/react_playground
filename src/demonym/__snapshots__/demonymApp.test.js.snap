// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DemonymApp /> Component renders on load 1`] = `
<div
  className="demonym_app"
>
  <CountrySelector
    countries={
      Array [
        Object {
          "name": "Barbados",
        },
        Object {
          "name": "Bahrain",
        },
      ]
    }
  />
  <Demonym
    country="Barbados"
    name="Barbadian"
  />
</div>
`;
